### PR TITLE
Action functions are not required to trigger while refreshing objects

### PR DESCRIPTION
**Pull request by nithinj**
* Server crash fix during refresh_node
* Crash fix during load conf
* Action functions are not required to trigger while refreshing object. run_rate improvement from 142 to 166.

**BUMMER!**
We had enabled syslogging for debugging purpose. It will print lots of messages to syslog in perf machines we test. As opposed to other logs which is being written to the SSD it goes to `/var/log/messages` which resides in HDD. It was spending a lot of time on I/O and hence plummeted figures we got.
It is giving standard results obtained for 19.1 branch after disabling it.
job_end_rate:  200.00/s
job_run_rate:  250.00/s

Results with 10k jobs with single server:
job_end_rate:  175.44/s
job_run_rate:  181.82/s

Results with 10k jobs with 3 servers:
job_end_rate:  212.77/s
job_run_rate:  217.39/s

### CODE REVIEW

**Comment by nithinj** [comment]
> find_queuebyname() might refresh an existing queue anyways, and since that is not going via the refresh_* functions it will still trigger the action functions.

find_queuebyname will invoke recov function and there we are deciding whether to trigger or not.

I believe what we are doing now is exactly what you are trying to explain.

**Comment by suresh-thelkar** [comment]
> > find_queuebyname() might refresh an existing queue anyways, and since that is not going via the refresh_* functions it will still trigger the action functions.
> 
> find_queuebyname will invoke recov function and there we are deciding whether to trigger or not.
> 
> I believe what we are doing now is exactly what you are trying to explain.

@nithinj, I agree with @subhasisb suggestion that we should remove the flag act_reqd as it can always be derived in functions db_to_svr_*(). This avoids passing the extra parameter to all these functions.

File: src/lib/Libifl/pbs_loadconf.c [hunk 1]
```diff
@@ -298,6 +298,16 @@ parse_psi(char * conf_value)
 		}
 		host = token;
 		if (*host == '\0') {
+			if (!pbs_conf.pbs_server_name) {
+				pbs_conf.pbs_server_name = malloc(PBS_MAXHOSTNAME);
+				*pbs_conf.pbs_server_name = '\0';
+			}
+			if (*pbs_conf.pbs_server_name == '\0') {
```
Comment (suresh-thelkar) [inline src/lib/Libifl/pbs_loadconf.c]: Can't we move the code under this if block to the earlier if block i.e. f (!pbs_conf.pbs_server_name) { } ?

Comment (nithinj) [reply src/lib/Libifl/pbs_loadconf.c]: done!

**Pull request merged**